core/lib/dr_tools.h

```diff
@@ -103,6 +103,15 @@ DR_API
 bool
 dr_using_all_private_caches(void);
 
+DR_API
+/**
+ * Returns false if DynamoRIO is being used as a regular standalone library.
```

Comment:
Examples would be nice (drdecode etc).

Reply:
Added a link to page_standalone in the docs (which talks about drdecode) and to dr_standalone_init().